docs/docs/dev_docs/getting_started/noir_contracts.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Setting up Aztec.nr Contracts
+---
+
+## Introduction
+
+This guide explains the necessary set up required to write your own contract using the Aztec.nr library and deploy it on the sandbox
+
+:::info Prerequisite reading
+If you haven't read [Aztec Sandbox](./sandbox.md) and [Noir](./noir.md), we recommend going there first.
+:::
+
+### Dependencies
+1. You will need the Noir build tool `nargo`, which you can install via [`noirup`](https://github.com/noir-lang/noirup). Make sure you install the `aztec` version of nargo:
+
+```bash
+curl -L https://raw.githubusercontent.com/noir-lang/noirup/main/install | bash
+noirup -v aztec
+```
+
+2. [Have a running sandbox and a code repo to interact with it](./sandbox.md)
+
+## Set up for aztec.nr contracts
+1. Inside the yarn project you created from the [Sandbox](./sandbox.md) page, create a sub-folder where the contracts will reside.
+```bash
+mkdir contracts
+```
+
+All contract projects will reside within this folder.
+
+2. Next, create a noir project using nargo by running the following in the terminal from the `contracts` folder
+```bash
+cd contracts
+nargo new example_contract
+```
+
+This creates a noir project with Nargo.toml (which is the manifest file of the project). Also, you will see a file in `example_contract/src/main.nr` which is where we will write our contract. 
+
+Your folder should look like:
+```
+.
+|-contracts
+| |--example_contract
+| |  |--src
+| |  |  |--main.nr
+|-src
+| |--index.ts
+```
+
+But before writing the contracts, we must add the aztec.nr library that adds smart contract syntax to Noir that the aztec sandbox can understand.
```

Comment:
Making tone a bit more authoritative by removing but. 
```suggestion
Before writing the contracts, we must add the aztec.nr library. This adds smart contract utility functions for interacting with the Aztec network.
```

consistency as refered to Aztec network at bottom of the page, which I prefer to sandbox. These docs should be independent of the target network